Robot_Engine/Compare/Properties/LoadingPanelProperty.cs

```diff
@@ -21,43 +21,41 @@
  */
 
 using RobotOM;
-using System.Collections.Generic;
 using BH.oM.Structure.Constraints;
-
+using System.Collections.Generic;
+using System;
+using BH.oM.Structure.SurfaceProperties;
 
 namespace BH.Engine.Robot
 {
-    public static partial class Convert
+    public class LoadingPanelPropertyComparer : IEqualityComparer<LoadingPanelProperty>
     {
         /***************************************************/
         /****           Public Methods                  ****/
         /***************************************************/
 
-        public static Constraint6DOF ToBHoMObject(this RobotNodeSupport robSupport)
+        public bool Equals(LoadingPanelProperty property1, LoadingPanelProperty property2)
         {
-            List<bool> fixity = new List<bool>();
-            RobotNodeSupportData suppData = robSupport.Data;
-            fixity.Add(suppData.UX != 0);
-            fixity.Add(suppData.UY != 0);
-            fixity.Add(suppData.UZ != 0);
-            fixity.Add(suppData.RX != 0);
-            fixity.Add(suppData.RY != 0);
-            fixity.Add(suppData.RZ != 0);
-
-            List<double> stiffness = new List<double>();
-            stiffness.Add(suppData.KX);
-            stiffness.Add(suppData.KY);
-            stiffness.Add(suppData.KZ);
-            stiffness.Add(suppData.HX);
-            stiffness.Add(suppData.HY);
-            stiffness.Add(suppData.HZ);
-
-            Constraint6DOF const6DOF = BH.Engine.Structure.Create.Constraint6DOF(robSupport.Name, fixity, stiffness);
-            return const6DOF;
+            if (property1.Name == property2.Name &&
```

Comment:
small thing, but would add null checks etc at the start of this. Can check the start of the `NodeDistanceComparer`